index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width" />
+	<link href="https://fonts.googleapis.com/css?family=Gelasio:400,500,600&display=swap" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Poppins:400,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+	<title>Mark &amp; Markup &vert; Full stack developer portfolio</title>
+</head>
+
+<body>
+	<header>
+		<div class="header-top">
+			<img src="./images/mamu-logo2.png" alt="Mark and Markup Logo" class="logo" />
+			<h1>Mark<span class="accent-color">&amp;</span>Markup</h1>
+			<img src="./images/hamburger-icon.png" alt="Menu Icon" id="burger-menu"/>
+		</div>
+		
+		<nav>
+
+			<div class="nav-links">
+				<ul>
+					<li><a href="#about">About</a></li>
+					<li><a href="#portfolio">Portfolio</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
+			</div>
+		</nav>
+	</header>
+	
+	<section class="showcase">
+		<div class="hero-caption">
+			<h1>Mark Eyles</h1>
+			<h3>Full stack developer portfolio</h3>
+		</div>
+		<a href="#" class="a-box">View this project ></a>
+	</section>
+	
+	<section class="about">
+		<a href="#" id="about" class="a-section-anchor">&nbsp;</a>	
+		<div class="center-container">
+			<a href="#about" class="a-section-title"><h2>About</h2></a>
+			<div class="about-content">
```

Comment:
could probably do with one less layer of div and put the center-container class on the div with about-content